src/components/TractionHeader.vue

```diff
@@ -23,6 +23,6 @@ import { bgColorClass } from '@/lib/BgColorClass'
 const environment = import.meta.env['VITE_ENVIRONMENT']
 
 const headerClasses = computed(() => {
```

Comment:
Im not sure this needs to be computed because its value will not change once its compiled.
Should just be able to have `const headerClasses = bgColorClass[environment]`